modules/virt-verify-status-bootsource-update.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-verify-status-bootsource-update_{context}"]
+= Verify status of automatic updating for a boot source
+
+You can verify the status of a boot source to confirm that automatic importing and updating is enabled or disabled.
+
+The `status` section of each boot sources listed in the `dataImportChronTemplates` of the `HyperConverged` CR indicates if automatic importing and updating is enabled or disabled. Examples of statuses include:
```

Comment:
I think we need to mention that this field is read-only, so the user won't try to modify boot resource by editing them here (it won't work, but let's be clear about it).

What do you think about the following?
```suggestion
The `status` section of each boot sources listed in the read-only `dataImportChronTemplates` field of the `HyperConverged` CR indicates if automatic importing and updating is enabled or disabled. Examples of statuses include:
```

Reply:
Works for me.